Cargo.toml

```
@@ -12,7 +12,8 @@ readme = "README.md"
 license = "MIT"
 
 [dependencies]
-bdk = { version = "^0.7", default-features = false, features = ["all-keys"]}
+#bdk = { version = "^0.7", default-features = false, features = ["all-keys"]}
+bdk = { git = "https://github.com/bitcoindevkit/bdk", barnch = "master", default-features = false, features = ["all-keys"]}
```

We should use the upcoming `^0.10` version when it's ready, prior to merging this PR  we can use the `release/0.10.0` branch which shouldn't change much before `0.10.0` is released: 
```suggestion
bdk = { git = "https://github.com/bitcoindevkit/bdk", branch = "release/0.10.0", default-features = false, features = ["all-keys"]}
```